pep_sphinx_extensions/pep_zero_generator/constants.py

```diff
@@ -45,4 +45,9 @@
 .. _specifications: https://packaging.python.org/en/latest/specifications/
 .. _PyPA specification update process: https://www.pypa.io/en/latest/specifications/#specification-update-process
 """,
+    "typing": """\
+Many recent PEPs propose changes to Python's static type system
+or otherwise relate to type annotations.
```

Comment:
```suggestion
Many recent PEPs propose additions or changes to type annotations
or otherwise relate to static typing.
```

I suggest this wording, since Python itself has type annotations, not a full in-built static type system (that's delegated to external tools), whereas these PEPs relate not just to type annotations themselves, but static typing more broadly, no?

Reply:
I prefer the current wording. The PEPs define a type system, which the external tools implement. Annotations are just one part of how the type system is reflected in code.